### ISSUE TITLE
ReferenceError: reporter is not defined

### ISSUE THREAD

**Issue by Smurf-IV** [ratherblue/eslint-html-reporter#4]
```
[11:33:57] ReferenceError: reporter is not defined
    at Gulp.<anonymous> (z:\NodeDev\NGen0.2\TestAndPackage\Gulpfile.js:67:23)
    at module.exports (z:\NodeDev\NGen0.2\TestAndPackage\node_modules\orchestrator\lib\runTask.js:34:7)
    at Gulp.Orchestrator._runTask (z:\NodeDev\NGen0.2\TestAndPackage\node_modules\orchestrator\index.js:273:3)
    at Gulp.Orchestrator._runStep (z:\NodeDev\NGen0.2\TestAndPackage\node_modules\orchestrator\index.js:214:10)
    at z:\NodeDev\NGen0.2\TestAndPackage\node_modules\orchestrator\index.js:279:18
    at finish (z:\NodeDev\NGen0.2\TestAndPackage\node_modules\orchestrator\lib\runTask.js:21:8)
    at z:\NodeDev\NGen0.2\TestAndPackage\node_modules\orchestrator\lib\runTask.js:43:4
    at process._tickDomainCallback (node.js:411:9)
```

from the following:
```
var report = require('eslint-html-reporter-ext');

gulp.task('lint', function () {
	var target = path.resolve(__dirname, '.lint');
	var stream = gulp.src(['./../Server/source/**/*.js'])
		.pipe(eslint({
			useEslintrc: true,
			configFile: "./.eslintrc"
		}))
		//.pipe(eslint.format("html", function (results) {
		.pipe(eslint.format(reporter, function (results) {
			fs.writeFileSync(target + '/lint-results.html', results);
		}));
```
Notes:
- Works okay if the default html reporter is used. (Commented out above)
- This report has been installed locally and also globally after it did not work the first time
- ext version also does not work.

**Comment by edendramis** [comment]
I believe you need to change `var report = ...` to `var reporter = ...` at the top there

**Comment by Smurf-IV** [comment]
School boy error etc.
Thanks for the spot.
